src/graphs.py

```diff
@@ -0,0 +1,222 @@
+from matplotlib import pyplot as plt
+from matplotlib import tri as tri
+from matplotlib import cm
+import scienceplots
+import numpy as np
+import os
+
+
+
+class GraphManager():
+    def __init__(self):
+        plt.style.use('science')
+    
+
+    def draw(self):
+        plt.show()
+        plt.close()
+
+    
+    def save_png(self, folder, file_name):
+        dir_path = os.path.dirname(os.path.dirname(__file__))
+        file_path = os.path.join(os.path.sep, dir_path, folder, file_name)
+        plt.savefig(file_path)
+        plt.close()
+
+    
+    def build_1D_compare(self, all_positions, sensor_positions, sensor_values, true_field, model_field):
+        # Draw the first plot
+        fig_1, (ax_1, ax_2) = plt.subplots(1,2, figsize=(18, 5))
+
+        ax_1.set_title('Field comparison')
+        ax_1.plot(all_positions, true_field, label='True field')
+        ax_1.plot(all_positions, model_field, label = 'Predicted field')
+        self.scatter_sensors(ax_1, sensor_positions, sensor_values)
+        ax_1.set_xlabel('x (m)')
+        ax_1.set_ylabel('field value (?)')
```

Comment:
I wanted to include units initially on the y-axis, only we don't know what the field value units are (it may be temperature or displacement or strain...). I put a '?' for the time being until the next person to write the code decides to force the user to specify the units.